thinc/tests/model/test_model.py

```diff
@@ -4,9 +4,10 @@
 import time
 from thinc.api import Adam, CupyOps, Dropout, Linear, Model, Relu
 from thinc.api import Shim, Softmax, chain, change_attr_values
-from thinc.api import concatenate, set_dropout_rate
-from thinc.api import use_ops, with_debug, wrap_model_recursive
-from thinc.compat import has_cupy_gpu
+from thinc.api import concatenate, set_dropout_rate, set_gpu_allocator
+from thinc.api import with_debug, wrap_model_recursive, use_ops
```

Comment:
Nitpick: why is this import list reordered? It was in alphabetic order?

Reply:
I think I removed `use_ops` at some point to try out another function and then re-added it to the end. Will fix.